src/lib/PermissionService.ts

```diff
@@ -36,6 +36,37 @@ class PermissionService {
       assignedByAaid: setter.aaid,
     });
   };
+
+  getNotes = async (): Promise<IUserNote[]> => {
+    return UserNote.findAll<UserNote>();
+  };
+
+  setNoteForUser = async (aaid: string, note: string, setter: ISessionUser): Promise<void> => {
+    (await UserNote.update(
+      {
+        note,
+        setByAaid: setter.aaid,
+      },
+      {
+        where: {
+          aaid,
+        },
+      },
+    ))[0] > 0 || // Check if a row was updated, otherwise create a new row
+      (await UserNote.create<UserNote>({
+        aaid,
+        note,
+        setByAaid: setter.aaid,
+      }));
+  };
+
+  removeUserNote = async (aaid: string): Promise<void> => {
```

Comment:
Absolutely not worried, but do you think we actually need to destroy them? Or just mark them blank?
Definitely not worth changing, just wondering if there was a reason for it

Reply:
Just in the interest of maintaining a specific pattern for this table. Where the row exists if the note exists, and there is one row/note per user.